README.md

```diff
@@ -1,14 +1,27 @@
 # speech2text
 
-This repo contains instructions for setting up and applying the speech2text app on Aalto Triton cluster. The app utilizes [WhisperX](https://github.com/m-bain/whisperX) automatic speech recognition tool and [Pyannote](https://huggingface.co/pyannote/speaker-diarization) speaker detection (diarization) pipeline. The speech recognition and diarization steps are run independently and their result segments are combined (aligned) using a simple algorithm which for each transcription segment finds the most overlapping (in time) speaker segment.
+>*_NOTE:_* The non-technical user guide for the Open On Demand web interface can be found [here](https://aaltorse.github.io/speech2text/).
+
+This repo contains instructions for setting up and applying the speech2text app on Aalto Triton cluster. The app utilizes
+
+- [WhisperX](https://github.com/m-bain/whisperX) automatic speech recognition tool
+- [wav2vec]() to find word start and end timestamps for WhisperX transcription
+- [Pyannote](https://huggingface.co/pyannote/speaker-diarization) speaker detection (diarization) tool 
+
+The speech recognition and diarization steps are run independently and their result segments are combined using a simple algorithm which for each transcription word segment finds the most overlapping (in time) speaker segment.
 
 The required models are described [here](#models). 
 
 Conda environment and Lmod setup is described [here](#setup). 
 
-Usage is describe [here](#usage).
+Command line (technical) usage on Triton is described [here](#usage).
+
+Open On Demand web interface (non-technical) usage is described [here](https://aaltorse.github.io/speech2text/)
```

Comment:
As the OOD is not part of the repo yet, let's remove this line and bring it back when OOD version is ready.